### ISSUE TITLE
Терминология и вычитка: бандлер/сборщик

### ISSUE THREAD

**Issue by another-guy** [reactjs/ru.reactjs.org#181]

**Comment by another-guy** [comment]
@gcor @ntishkevich @Heegiiny @avevlad есть соображения, как лучше переводить?

"Сборщик" -- хорошее русское слово, но лично мне не ясно, насколько оно точное. Склоняюсь к тому, что для тех, кто читает впервые, _сборщик_ попонятнее, чем _бандлер_.

**Comment by gcor** [comment]
У нас уже есть в терминологии bundler = бандлер, но меня смущают слова, которые встречаются: bundle, bundling, bundled

И их бы я точно не переводил как бандлирование файлов или бандлованные файлы.
Я за сбоку, сборщик, собирание и собранные.

**Comment by avevlad** [comment]
`bundling` как бандлирование переводить точно не надо


`bundling` бандлинг

**Comment by avevlad** [comment]
`bundler` -> сборщик
`bundling` -> сборка

`bundle file` -> ?

@another-guy Я согласен с тем что сборщик попонятнее, чем бандлер. Но непонятно что делать с "bundle" files. 

В случает перевода Code Splitting пока предлагаю оставить все как есть, только поправить определение **Бандлинга**:

с

> Бандлинг — это процесс отслеживания импортированных файлов и их объединения их в один «бандл-файл».

на

> Бандлинг — это процесс **сборки** файлов с модулями и их зависимостями в один «бандл-файл».

**Comment by another-guy** [comment]
@gcor, да, я знаю, что у нас в словаре _бандлер_. Но в ряде мест перевели как _сборщик_, а хотелось бы, чтобы перевод был последователен.

@avevlad 
bundler -> сборщик  👍 
bundling -> сборка  👍 
bundle file -> **собранный файл** ❓  (в отличие от _файл сборки_, который допускает двусмысленное толкование)

И тогда можно так написать:
> Сборка (или «бандлинг») — это процесс выявления импортированных файлов и их объединения их в один «собранный» файл (часто называемый «bundle» или «бандл»).

**Comment by avevlad** [comment]
Этот **файл сборки** после подключения на веб-страницу загружает всё приложение за один раз.

----------

**Бандлинг** это хорошо, но по мере роста вашего приложения, ваш бандл также будет расти

**Cборка** это хорошо, но по мере роста вашего приложения, ваш (**собранный файл** | **файл сборки**) также будет расти

-----------

У них есть хорошее руководство по разделению **бандла** с серверным рендерингом.

У них есть хорошее руководство по разделению **файла сборки** с серверным рендерингом.

-----------


Разделение кода это возможность которая поддерживается такими **бандлерами** как Webpack или Browserify (с factor-bundle), она может разделить ваш **бандл** на несколько кусочков и загружать их по мере необходимости.

Разделение кода это возможность которая поддерживается такими **сборщиками**  как Webpack или Browserify (с factor-bundle), она может разделить ваш **файл сборки** на несколько кусочков и загружать их по мере необходимости.

-------------

Она автоматически загрузит **бандл** содержащий OtherComponent, когда этот компонент будет отрендерен.


Она автоматически загрузит **собранный файл**  содержащий OtherComponent, когда этот компонент будет отрендерен.



-----------------
----------------

Сложно, может я просто привык, но мне бандл файл как-то понятнее.

**Comment by avevlad** [comment]
> Сборка (или «бандлинг») — это процесс выявления импортированных файлов и их объединения их в один «собранный» файл (часто называемый «bundle» или «бандл»).

Да, так клево, постараюсь сегодня (по MSK) успеть обновить #246

**Comment by another-guy** [comment]
@gaearon каким путём идти? У нас и _бандлер_ и _сборщик_ есть в разных местах. Хотелось бы всё же определиться и сделать в одном стиле.

В этом issue, вроде бы, сходимся на том, что "сборщик" более "по-русски", но git blame мне сказал, что в словарь _бандлер_ добавил ты и мне как-то некомфортно идти против. Проясни, пожалуйста, и можно будет сварить pull request для этой долгоиграющей issue.
